aws-ebs-csi-driver/templates/manifest.yaml

```diff
@@ -214,10 +214,13 @@ spec:
         - name: ebs-plugin
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           args:
+            {{- if .Values.driverMode }}
+            - {{ .Values.driverMode }}
```

Comment:
How about just hard code `controller` here? since it always going to be controller service. And, for the sake of completeness, could you update the daemonset to use the `node` subcommand to always start it as node service?